news/_posts/2021-12-14-prestashop-beyond-1-7.md

```diff
@@ -0,0 +1,101 @@
+
+In addition, **maintaining a software version without introducing backward-incompatible changes is hard**, and it gets worse with time. PrestaShop doesn't exist in the void, and our ecosystem has an agenda of its own. With new PHP versions being released (and abandoned) every year, libraries are forced to evolve quickly to keep up. Symfony 3.4 has already reached its end of life. PrestaShop cannot risk using obsolete, abandoned libraries, so it needs to keep up as well.
+
+As we said last year, we think that one of the key challenges for achieving PrestaShop's technical transformation is about finding the right pace, so that it can be delivered both progressively and predictably. We believe that the best answer for that is to steer the project towards an iterative cycle of **smaller, more focused, more frequent releases** — each building on the work laid out by the previous one.
+
+## Introducing a new release roadmap
+
+The [Semantic Versioning convention](https://build.prestashop.com/news/a-more-semantic-versioning-scheme/) (SemVer) only allows introducing backward-incompatible changes in major versions. This means that in order to boost PrestaShop's technical transformation, major versions need to be published more often than once every five years or more. At the same time, the community needs time to adapt to changes, as they carry a cost for merchants and developers alike. They need visibility to anticipate **what** kind of changes to expect, **when** will they encounter them, and finally, **how long** will their investment last until they need to adapt again.
+
+This requires a delicate balance: change too slowly, and project becomes obsolete; change too quickly, and it becomes too expensive for the community to keep up. With that in mind, here is our proposal.
+
+The new release roadmap will adhere more closely to the SemVer definition: a version is considered "major" when it introduces breaking changes, while minor versions only add features in a retrocompatible way. By following these principles strictly, upgrading from a major version to the minor version following it should be as easy as upgrading to a patch version, as everything is expected to work the same way. In this sense, it also means that a major version doesn't necessarily mean _big_ changes, just _some_ changes.
+
+Starting on our next release, PrestaShop will begin alternating between major versions, focusing on refactoring and technology advancements, and minor versions, emphasizing on new or improved features. This means that **the next PrestaShop release will be a major version**, which will be followed by a minor version, then a major version, then a minor version, and so on.
+
+Our target is to be able to publish a release every 6–9 months. Since every other release will be a major version, the community will be able to expect a new major version every 12 to 18 months.
```

Comment:
```suggestion
The target is to be able to publish a release every 6–9 months. Since every other release will be a major version, the community will be able to expect a new major version every 12 to 18 months.
```